BCStudentSoftwareDevTeam/cas
# Handling cancel image upload 373

Cancelling or removing image removes the image from the peewee database and deletes the file from the folder. Fixes the issue where you have to click remove, refresh the page, then click remove again for the file to be removed. Also actually deletes the file, whereas it would previously just be hidden. Cancelling also works. 

Added multipart/form-data to buildingManagement.html
Used same JS function method for either removing or cancelling. 
Added os remove function to delete the file. 
Resets roomImageURL to null.

How to test: 
Upload an image and remove it, You may refresh the page and check the database and file directory. 
Or, upload a large file and cancel it before it uploads completely.
